### PR TITLE
Updated README.md Grammar

### PR DESCRIPTION
**Pull request by KukretiShubham**
Suggested appropriate changes in grammar.

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ analysis / manipulation tool available in any language**. It is already well on
 its way towards this goal.
 
 ## Main Features
-Here are just a few of the things that pandas does well:
+Here are just a few of the things that pandas do well:
```
Comment (jbrockmendel) [inline README.md:32]: In this context "pandas" is not a plural noun.

**Comment by attack68** [comment]
some of these changes were already suggested in #42454. It was closed, however, since those/these changes are not grammatical errors, they are more subjective preferences of the author and changes are not needed.

**Comment by MarcoGorelli** [comment]
Agree that these changes are not needed - please see [good first issues](https://github.com/pandas-dev/pandas/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [contributing guide](https://pandas.pydata.org/docs/development/contributing.html) for how to contribute to pandas

**Review by KukretiShubham — COMMENTED** [review]
> some of these changes were already suggested in #42454. It was closed, however, since those/these changes are not grammatical errors, they are more subjective preferences of the author and changes are not needed.


Thank you for reviewing. Will look for the existing issue now.

**Pull request closed without merging**